src/Uno.UI.Runtime.Skia.Wpf/Extensions/UI/Xaml/Controls/TextBoxViewExtension.cs

```diff
@@ -17,11 +15,17 @@ internal class TextBoxViewExtension : ITextBoxViewExtension
 	{
 		private readonly TextBoxView _owner;
 		private ContentControl? _contentElement;
-		private WpfTextViewTextBox? _currentInputWidget;
+
+		private WpfTextViewTextBox? _currentTextBoxInputWidget;
+		private System.Windows.Controls.PasswordBox? _currentPasswordBoxInputWidget;
```

Comment:
PasswordBox is sealed in WPF. So I couldn't a new type similar to `WpfTextViewTextBox`